src/openforms/authentication/contrib/eherkenning/views.py

```diff
@@ -126,6 +127,17 @@ def get(self, request):
                 "urn:etoegang:1.9:EntityConcernedID:Pseudo"
             ]
 
+        # Extract the branch number service restriction(s) - this is all super vague and
+        # we don't seem to have proper test accounts for this...
+        # See https://afsprakenstelsel.etoegang.nl/Startpagina/v2/interface-specifications-dv-hm,
+        # section "AttributeStatement" for an example response.
+        # This translates to a list of strings (12 chars, all digits)
+        if branch_numbers := attributes.get(
+            "urn:etoegang:1.9:ServiceRestriction:Vestigingsnr"
+        ):
+            logger.info("Got branch numbers: %r", branch_numbers)
+            request.session[EHERKENNING_BRANCH_NUMBERS_SESSION_KEY] = branch_numbers
```

Comment:
this is pretty nasty and a setup similar to the OIDC variants where everything is scoped under a single session key would be better